packages/ripple-nuxt-tide/lib/core/tide.js

```diff
@@ -443,6 +448,33 @@ export const tide = (axios, site, config) => ({
     return pageData
   },
 
+  getPageByShareLink: async function (path, section) {
```

Comment:
@alan-cole Can we split this function to different file? I would like to make tide.js only focus on the basic Tide API communication. Thanks!

Reply:
I'll create a submodule for sharelinks and add this in.